src/main/java/jeju/controller/TourRestController.java

```diff
@@ -18,14 +18,55 @@ public class TourRestController {
 	@Autowired
 	private TourDao tourDao;
 	
-	
 	@GetMapping("/tour/view")
-	public List<TourDto> getAllTour(
-			@RequestParam(defaultValue = "0") int start,
-            @RequestParam(defaultValue = "8") int perpage
-			)
-	{
-		return tourDao.getAllTour(start, perpage);
+	public HashMap<String, Object> getAllTour(@RequestParam String word, @RequestParam (defaultValue = "1") int currentPage) {
+		HashMap<String, Object> resultMap = new HashMap<String, Object>();
+		
+		//페이징처리
+		//페이징에 처리에 필요한 변수들
+		int perPage=12; //한페이지당 보여지는 게시글의 갯수
+		int totalCount=0; //총 개시글의 개수
+		int totalPage;//총페이지수
```

Comment:
변수를 선언할 때 이왕이면 초기값을 지정해주시는 게 오류 확률도 줄이고 좋지 않을까 싶어요!

```suggestion
		int totalPage = 0;//총페이지수
```